src/Faker/Provider/cs_CZ/Person.php

```diff
@@ -425,6 +425,117 @@ class Person extends \Faker\Provider\Person
         'Bc.', 'Ing.', 'MUDr.', 'MVDr.', 'Mgr.', 'JUDr.', 'PhDr.', 'RNDr.', 'doc.', 'Dr.'
     );
 
+    protected static $birthNumbers = array(
```

Comment:
It is personal identification number of every citizen of the Czech
Republic. It has to met some requirements. More info here:
http://en.wikipedia.org/wiki/National_identification_number#Czech_Republic.2FSlovakia
It could be generated but I was too lazy to write a generator so I've used
someone's else.

On Wed, Mar 18, 2015, 16:35 Francois Zaninotto <notifications@github.com>
wrote:

> In src/Faker/Provider/cs_CZ/Person.php
> <https://github.com/fzaninotto/Faker/pull/535#discussion_r26674279>:
>
> > @@ -425,6 +425,117 @@ class Person extends \Faker\Provider\Person
> >          'Bc.', 'Ing.', 'MUDr.', 'MVDr.', 'Mgr.', 'JUDr.', 'PhDr.', 'RNDr.', 'doc.', 'Dr.'
> >      );
> >
> > +    protected static $birthNumbers = array(
>
> Why these numbers and not others? Why doesn't it rely on a numerify()
> pattern?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fzaninotto/Faker/pull/535/files#r26674279>.
>